examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "compliant-eligibility",
+   "metadata": {},
+   "source": [
+    "# Complex data type detection using the Data Labeler component of the Data Profiler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "light-robert",
+   "metadata": {},
+   "source": [
+    "In this example, we utilize the Data Labeler component of the Data Profiler to detect the sensitive information for both structured and unstructured data. In addition, we show how to train Data Labeler on some specific dataset with different list of entities.\n",
+    "\n",
+    "First, let's import the libraries needed for this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ethical-criterion",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, '../..')\n",
+    "import os\n",
+    "import dataprofiler as dp\n",
+    "import json\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "configured-basis",
+   "metadata": {},
+   "source": [
+    "## Structured data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "recorded-storage",
+   "metadata": {},
+   "source": [
+    "We use the aws honeypot dataset in the test folder for this example. First, look at the data using Data Reader class of the Data Profiler. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "preceding-protection",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = dp.Data(\"../../dataprofiler/tests/data/csv/aws_honeypot_marx_geo.csv\")\n",
```

Comment:
should we assume they are cloning the repo or just executing the examples? In the latter case should we have a set of example datasets we utilize for these examples?

Reply:
I discussed this with Austin. Looks like we don't need a separate data folder if the datasets are the same as in the test folder.